src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/PostgresLedgerExtenderService.cs

```diff
@@ -556,7 +575,10 @@ private async Task<ExtendLedgerReport> ProcessTransactions(ReadWriteDbContext db
                         ErrorMessage = committedTransaction.Receipt.ErrorMessage,
                         Output = committedTransaction.Receipt.Output != null ? JsonConvert.SerializeObject(committedTransaction.Receipt.Output) : null,
                         NextEpoch = committedTransaction.Receipt.NextEpoch?.ToJson(),
-                        Events = committedTransaction.Receipt.Events != null ? JsonConvert.SerializeObject(committedTransaction.Receipt.Events) : null,
+                        EventsSbor = eventsTuple?.Select(x => x.data).ToArray(),
+                        EventsSchemaHash = eventsTuple?.Select(x => x.schemaHash).ToArray(),
+                        EventsTypeIndex = eventsTuple?.Select(x => x.typeIndex).ToArray(),
+                        EventsTypeKind = eventsTuple?.Select(x => x.typeKind).ToArray(),
```

Comment:
1. What's the point of this `eventsTuple` if we effectively select/copy everything we collected first? Can't we just just inline all of that in here?
2. Why do we support nullable collections? Wouldn't empty collection be more appropriate data type?